examples/static/js/style-defaults.js

```diff
@@ -0,0 +1,6 @@
+const originalStyleDefault = { ...bpmnvisu.StyleDefault };
+const resetStyleDefault = () => {
+  for (let key of Object.keys(bpmnvisu.StyleDefault)) {
+    bpmnvisu.StyleDefault[key] = originalStyleDefault[key];
+  }
```

Comment:
**thought:** this is the only implementation I've managed to get to work.
For example, the following has no effect. For example, the following has no effect. There's probably a good reason for this, but I haven't looked into it, because I've found another solution.
```js
bpmnvisu.StyleDefault = { ...bpmnvisu.StyleDefault };
```

Reply:
It's normal to have no effect, if we set a variable with its own values 😄